bin/tcp_server.ml

```diff
@@ -0,0 +1,65 @@
+open Lwt
+
+(* Shared mutable counter *)
+let counter = ref 0
+
+let listen_address = Unix.inet_addr_loopback
+let port = 9000
+let backlog = 10
+
+let string_to_char s =
+    char_of_int @@ int_of_string s
+
+let handle_message msg =
+    match msg with
+    | "exit" -> "Bye!"
+    | _ ->
+    let param_list = Str.split_delim (Str.regexp " ") msg in
+    match param_list with
+    | [_;_;_] -> let char_param_list = List.map string_to_char param_list in
+    Client.play_note (Array.of_list char_param_list) ();
```

Comment:
Try to get `ocp-indent` or `ocamlformat` working - there should be more indentation here